### PR TITLE
change logging level + move vars to default

### PR DESCRIPTION
**Pull request by tokyowizard**
This PR allows to change the default logging level, `INFO`, to [user assigned level](http://logging.apache.org/log4j/1.2/manual.html#Logger_hierarchy).

The set of possible levels, that is:

`TRACE`,
`DEBUG`,
`INFO`,
`WARN`,
`ERROR` and
`FATAL`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,6 +26,10 @@ node3.zookeeper.somedc.prod1  ansible_ssh_host=172.16.0.103 zoo_id=3
 [zookeeper1-somedc-prod:vars]
 # Must be set to the name of the subgroup
 zookeeper_cluster_name = zookeeper1-somedc-prod
+
+# Optional. Change the logging level.
```
Comment (MiLk) [inline README.md:30]: Could you add a link to the documentation with the list of valid values?

File: tasks/centos.yml [hunk 2]
```diff
@@ -39,3 +39,16 @@
...
+
+- name: "Set logging level"
+  lineinfile: >
+    dest=/etc/zookeeper/conf/log4j.properties
+    regexp='{{ item.regex }}'
+    line='{{ item.line }}'
+  with_items:
+    - { regex: "^zookeeper.root.logger=", line: "zookeeper.root.logger={{ zookeeper_log_level }}, CONSOLE" }
+    - { regex: "^zookeeper.console.threshold=", line: "zookeeper.console.threshold={{ zookeeper_log_level }}" }
+  when: ansible_distribution_major_version|int >= 7
```
Comment (MiLk) [inline tasks/centos.yml:77]: If these options are defined here, maybe they should be removed from `/etc/zookeeper/conf/zoo.cfg`.

**Review by MiLk — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (tokyowizard) [reply README.md:30]: Thanks! I've added the link just below that line.

File: tasks/centos.yml (hunk 2 above)
Comment (tokyowizard) [reply tasks/centos.yml:77]: Thanks! Good idea. I've removed it from `zoo.cfg`

Comment (MiLk) [reply tasks/centos.yml:77]: Hum, they should be kept for Debian and Centos 6, no?

**Review by MiLk — CHANGES_REQUESTED** [review]

File: tasks/centos.yml (hunk 2 above)
Comment (tokyowizard) [reply tasks/centos.yml:77]: Fixed for CentOS 6.

File: tasks/debian.yml [hunk 3]
```diff
@@ -1,30 +1 @@
...
-  copy: >
-    src=debian-zookeeper-client
-    dest=/usr/bin/zookeeper-client
-    owner=root
-    group=root
-    mode=0755
-  tags:
-    - zookeeper
-    - files
+- fail: msg="The ZooKeeper Playbook does not support Ubuntu"
```
Comment (MiLk) [inline tasks/debian.yml]: Debian

**Review by MiLk — CHANGES_REQUESTED** [review]

File: tasks/debian.yml (hunk 3 above)
Comment (tokyowizard) [reply tasks/debian.yml]: Thanks. fixed.

**Review by MiLk — APPROVED** [review]

**Pull request merged**